### DISCUSSION TITLE
Deploy "Teleport Windows Desktop Service" with teleport-cluster Helm chart

### DISCUSSION THREAD

**Discussion by fsantoso82** [gravitational/teleport#43182] [Q&A] [answered]
Hi Guys,

I try to deploy an Teleport Cluster via Helm Charts from the repo "https://charts.releases.teleport.dev".
This works great.

But now I try to figure out how to get the "Windows Desktop Service" to deploy with the helm chart.
Or is it not implemented?

If not how can I deploy a container in my kubernetes Cluster without providing an extra Linux VM with the Windows Desktop Service installed?

Maybe some of you already figured that out, as I didn´t find anythin on the internet.

Thx in advance.

**Comment by webvictim** [comment, marked as answer]
Not currently implemented - see https://github.com/gravitational/teleport/issues/15956
